sql/protocol_classic.cc

```diff
@@ -3108,10 +3108,27 @@ bool Protocol_classic::start_result_metadata(uint num_cols_arg, uint flags,
     Set it to NULL if we skip resultset metadata to avoid
     ::storeXXX() method's asserts failures.
   */
-  if (m_thd->variables.resultset_metadata == RESULTSET_METADATA_FULL)
+  if (m_thd->variables.resultset_metadata == RESULTSET_METADATA_FULL) {
+#ifdef WITH_WSREP
```

Comment:
Can we have it this way?
```suggestion
  if (m_thd->variables.resultset_metadata == RESULTSET_METADATA_FULL)
#ifdef WITH_WSREP
  {
```

Reply:
sure, original code does not had curly brackets. Fixed